src/pages/HH_ManageHH.page

```diff
@@ -1,11 +1,11 @@
 <apex:page controller="HH_ManageHH_CTRL" standardStylesheets="true" showHeader="true" docType="html-5.0" >
     <html xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
         <apex:includeLightning />
-        <apex:stylesheet value="{!URLFOR($Resource.sfdo203, '/assets/styles/salesforce-lightning-design-system-vf.min.css')}" />
         <apex:stylesheet value="{!URLFOR($Resource.CumulusStaticResources, '/npsp-slds/npsp-common.css')}" />
         <apex:includeScript value="{!URLFOR($Resource.CumulusStaticResources, '/jquery/jquery-2.2.4.js')}"/>
-        
-        <style>       
+        <apex:slds />
+
+        <style>
```

Comment:
I see that this is a pre-existing style class name, but to avoid conflicts with anything else, I typically like to use a prefix or something that makes the class name unique, such as `npsp-required`. Unless maybe this is overriding a Salesforce class name.

Reply:
Good idea, but yea this is for the ui:input required css.